static/js/portico/signup.js

```diff
@@ -198,6 +210,15 @@ $(() => {
     $("#source_realm_select").on("change", update_full_name_section);
     update_full_name_section();
 
+    $("#realm_with_subdomain")
+        .siblings("#id_team_subdomain")
+        .on("input", () => {
+            if (!$("#realm_with_subdomain").is(":checked")) {
+                $("#realm_with_subdomain").prop("checked", true);
+                subdomain_required(true);
```

Comment:
I am wondering if the above two lines could be replaced with `$("#realm_with_subdomain").trigger("click")`.

Reply:
Updated. Thanks.